move/its/sources/service.move

```diff
@@ -30,14 +30,10 @@ module its::service {
     const MESSAGE_TYPE_SEND_TO_HUB: u256 = 3;
     const MESSAGE_TYPE_RECEIVE_FROM_HUB: u256 = 4;
 
-    /**
-     * @dev Chain name for Axelar. This is used for routing ITS calls via ITS hub on Axelar.
-     */
+    // Chain name for Axelar. This is used for routing ITS calls via ITS hub on Axelar.
     const ITS_HUB_CHAIN_NAME: vector<u8> = b"Axelarnet";
 
-    /**
-     * @dev Identifier to be used as destination address for chains that route to hub. For Sui this will probably be every supported chain.
-     */
+    //Identifier to be used as destination address for chains that route to hub. For Sui this will probably be every supported chain.
```

Comment:
```suggestion
    // Identifier to be used as destination address for chains that route to hub. For Sui this will probably be every supported chain.
```